### PR TITLE
Feature integra 1027

### PR DESCRIPTION
**Pull request by thomasalbertanaplan**
@bdmaven updated for INTEGRA-1027 Document Mulesoft configuration change for 1.1.5 the Guide in both PDF and .adoc

### CODE REVIEW

File: doc/AnaplanConnectorforMuleSoft-Guide.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ anaplan.deleteActionId=
 == Setting up the Environment
 
 1. Install the following:
-- Java 7 
+- Java 7 SDK (if installing through Anypoint Studio or GitHub), but if installing through Anypoint Studio, Java 8 SDK is also supported.
```
Comment (shuvozula) [inline doc/AnaplanConnectorforMuleSoft-Guide.adoc]: Should be reworded as: "Java7 SDK if installing connector from source in Anypoint Studio, else Java8 SDK if installing from Mulesoft Connector repository."

File: doc/AnaplanConnectorforMuleSoft-Guide.adoc [hunk 2]
```diff
@@ -115,13 +115,18 @@ Note: Your version of Anypoint Studio might be slightly different than what is d
...
-1. In Anypoint Studio, on the File menu, click Import, and select Anypoint Studio Project from External Location.
-2. In the Mule Import dialog, navigate to the Project root of one of the demo projects, such as anaplan-export, and make sure that Copy project into workspace is selected.
-3. You can now install another demo project. There are three demo projects, one for delete, one for import, and one for export.
+1.  (Recommended for most users) - download the demo.zip file from the link at https://community.anaplan.com/web/guest-staging/anapedia/integrations/third-party-data-integration/anaplan-connector-for-mulesoft
+    or
+    (If you are a software developer experienced with GitHub) - they are inside the "Demo" folder that is part of the downloadable zip in GitHub. 
+2. In Anypoint Studio, on the File menu, click Import, and select Anypoint Studio Project from External Location.
+3. In the Mule Import dialog, navigate to the Project root of one of the demo projects, such as anaplan-export, and make sure that Copy project into workspace is selected.
+4. You can now install another demo project. There are three demo projects, one for delete, one for import, and one for export.
```
Comment (shuvozula) [inline doc/AnaplanConnectorforMuleSoft-Guide.adoc]: Step-4 seems a bit out of place. Lets break up the step and add "There are three demo projects, one for delete, one for import, and one for export." and insert as Step-2 above. 

"You can now install another demo project by repeating the above steps" as the _last_ step.

**Comment by shuvozula** [comment]
In the .adoc as well as .pdf files, please remove the column-separator and delimiter references. Add note that these are set in the Import action (Datasource definition precisely) and if the user needs to modify them, they go back to the Anaplan UI and fix it there.

**Comment by thomasalbertanaplan** [comment]
Updated with developer's feedback

**Comment by thomasalbertanaplan** [comment]
Fixed a minor typographical error in the latest PDF.

**Pull request merged**